src/rgw/rgw_op.cc

```diff
@@ -5559,6 +5559,28 @@ RGWBulkUploadOp::parse_path(const boost::string_ref& path)
   return boost::none;
 }
 
+boost::optional<std::pair<std::string, std::string>>
```

Comment:
I don't think we really need the `boost::optional` here especially are as we're dereferencing it without the check for emptiness.